ExtractData/PKHeX.ExtractData/Program.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+
+namespace PKHeX.ExtractData
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string folder = Path.GetDirectoryName(System.Reflection.Assembly.GetExecutingAssembly().Location);
+            if (Path.GetFileName(folder).ToLower() == "debug")
+                folder = Directory.GetParent(folder).FullName;
+            if (Path.GetFileName(folder).ToLower() == "release")
+                folder = Directory.GetParent(folder).FullName;
+            if (Path.GetFileName(folder).ToLower() == "bin")
+                folder = Directory.GetParent(folder).FullName;
+            if (!Directory.Exists(folder)) return;
```

Comment:
I don't think line 21 is needed.  The working directory is going to exist 9 times out of 10.

If it doesn't, it would be better to throw an exception, or since this is a console app, write to the console.